src/js/lib/content/src/index.js

```diff
@@ -179,20 +179,37 @@ const MAX_MESSAGE_LENGTH = 500;
 
 function remoteLog(event,data) {
```

Comment:
I also have question to the line above (doesn't need to be fixed in this PR, but Id love to know the answer). Is Mixpanel synchronous initialization the only option? (I hope not!). If the internet connection is slow, the whole GUI freezes on Mixpanel initialization :( We should instead buffer logs and send them after the initialization is done. Please either fix it or create issue for it.

Reply:
Let's create an issue. I have not researched this deeply, but what we do here is the way described in the docs. At first, glance there seems to be no async alternative, so we'd have to wrap this ourselves.

Reply:
I was browsing Mixpanel docs and it seems that their API should be async (with callbacks). So Im wondering why it happens. Anyway, it needs to be investigated, that's right. A hint - in chrome dev tools you can limit the internet speed per tab to see how it works with a low speed! Wpuld you be so nice and create issue for it, please?

Reply:
https://github.com/enso-org/ide/issues/1280